### DISCUSSION TITLE
how to convert cnn outpt into yolov8 for further process

### DISCUSSION THREAD

**Discussion by sagarmane219** [community/community#137386] [Programming Help]
### Select Topic Area

Question

### Body

HI Team,

If any one knows, how to convert cnn output into yolov8 for further process.

Please suggest.

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @sagarmane219 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
